src/WeSay.Project.Tests/WeSayWordsProjectTests.cs

```diff
@@ -1114,5 +1116,136 @@ public void LoadFromLiftLexiconPath_FolderAndLiftFileNameDifferInCase_PathToLift
 				Assert.That(project.PathToLiftFile, Is.EqualTo(e.PathToLiftFile));
 			}
 		}
+
+		private void OnProject_MeaningFieldChanged(object sender, WeSayWordsProject.StringPair pair)
+		{
+			var project = sender as WeSayWordsProject;
+			foreach (var task in project.TaskConfigurations)
+			{
+				var mf = task as ICareThatMeaningFieldChanged;
+				if (null == mf)
+					continue;
+				mf.OnMeaningFieldChanged(pair.from, pair.to);
+			}
+		}
+
+		private void VerifyTasks(IEnumerable<ITaskConfiguration> taskconfigs, string meaningField)
+		{
+			// want to assert the following changes
+			// dictionary task meaningfield
+			// MissingInfo tasks have changed
+			bool dict = false, meanings = false, pos = false, exsent = false;
+			Assert.NotNull(taskconfigs);
+			foreach (object task in taskconfigs)
+			{
+				Assert.NotNull(task);
+
+				if (null != task as DictionaryBrowseAndEditConfiguration)
```

Comment:
I would use the same pattern here that you used for MissingInfoConfiguration. I think it is clearer.